ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/services/transaction/TransactionFiltration.scala

```diff
@@ -48,26 +48,36 @@ object TransactionFiltration {
           InvertedTransactionFilter
             .extractFrom(transactionFilter))
 
+      println(s"PartiesByTemplate: $partiesByTemplate")
```

Comment:
friendly reminder to remove `println`s. here and below. I tend to forget them sometimes..

Reply:
yikes, thank you.